src/docs/010_manual/60_further_reading.adoc

```diff
@@ -6,102 +6,100 @@ ifndef::imagesdir[:imagesdir: ../images]
 
 include::../_feedback.adoc[]
 
-This chapter lists some additional references to interesting resources.
+Everything you need to know about docToolchain, from the underlying technology to useful resources.
 
 == Underlying Technologies
 
-This project relies on many underlying technologies.
-Here is a lists and some explaining words
+DocToolchain uses many underlying technologies. Explore the list with some resources.
```

Comment:
You could shorten this sentence to the following and move it up to the Introduction section and replace the sentence currently there. It says it all:

Learn more about the technologies that support Doc Toolchain, as well as some useful resources.